### DISCUSSION TITLE
module.exports issue

### DISCUSSION THREAD

**Discussion by rogerl50** [peggyjs/peggy#227] [Q&A] [answered]
In my application, all the js files are concatenated on the client, so that for example all the peggy symbols are globally available. I can't figure out how to just prevent the `module.export` section from being generated.

Ideally, what I'd like would be to refer to `peg$Parse` and `peg$SyntaxError` by a custom name, but if what I get is that I have to use the existing names, I'm fine with that. (Note: I am by no means a js expert, so I won't understand anything complicated involving module structures and the like.)

**Comment by hildjj** [comment, marked as answer]
Have you tried `--format globals --export-var MyGlobalName`?
